cmd/argo/commands/resume.go

```diff
@@ -2,14 +2,25 @@ package commands
 
 import (
 	"fmt"
+	"k8s.io/apimachinery/pkg/fields"
 	"log"
 	"os"
 
 	"github.com/argoproj/argo/workflow/util"
 	"github.com/spf13/cobra"
 )
 
+type resumeOps struct {
+	stepFail           bool     // --step-fail
```

Comment:
I'm not sure about using the "step" term, we use the term "node" in the code. @simster7 ?

Reply:
@sarabala1979 @jessesuen thoughts on this naming?

Reply:
I favor "node" as well